data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,802 @@
+	category "Turrets"
+	cost 120000
+	thumbnail "outfit/avgi green laser"
+	"mass" 6
+	"outfit space" -6
+	"weapon capacity" -6
+	"turret mounts" -1
+	"optical jamming" 6
+	"lasing power" -1
+	weapon
+		"hardpoint sprite" "hardpoint/avgi green laser"
+		sound "disruptor"
+		"die effect" "laser ejecta" 4
+		"die effect" "green laser reflection" 30
+		"die effect" "green laser scattering"
+		"fire effect" "green laser fire"
+		"anti-missile" 2
+		"velocity" 1800
+		"lifetime" 1
+		"firing energy" 9
+		"firing heat" 9
+	description "This lightweight turret can be fed a beam of light from a lasing generator, which is then combined and focused to produce an intense beam suitable for targeting and destroying missiles. Unlike a typical pulsed-shot human antimissile system, this turret seems designed to hold a beam on target at an extreme distance for a longer period of time using its precise optics, making it better suited for defending and supporting other ships from afar."
+	description "The turret itself only contains the optics needed to synthesize, focus, and target the beam, allowing the laser light to be produced elsewhere. Because of this, an Optical Lasing Generator is needed to run this turret, providing enough power to support three Green Lasers."
+
+
+outfit "Blue Optical Laser"
+	category "Turrets"
+	licenses
+		"Twilight Guard"
+	cost 120000
+	thumbnail "outfit/avgi blue laser"
+	"mass" 9
+	"outfit space" -9
+	"weapon capacity" -9
+	"turret mounts" -1
+	"optical jamming" 12
+	"lasing power" -3
+	weapon
+		"hardpoint sprite" "hardpoint/avgi blue laser"
+		sprite "projectile/avgi laser blue"
+			"frame rate" 1
+		sound "disruptor"
+		"hit effect" "laser ejecta" 4
+		"hit effect" "blue laser reflection" 30
+		"hit effect" "blue laser scattering"
+		"fire effect" "blue laser fire"
+		"velocity" 2400
+		"lifetime" 1
+		"reload" 1
+		"turret turn" 6
+		"firing energy" 12
+		"firing heat" 12
+		"shield damage" 3
+		"hull damage" 4.2
+		"heat damage" 6
+		"damage dropoff" 0 3600
+	description "This lightweight turret can be fed a beam of light from a lasing generator, which is then combined and focused to produce an intense beam ideal for damaging other ships. By pulsing thousands of times each second, the beam can create thermal and mechanical shocks in the target material, enhancing its ability to drill through armor. Unlike the advanced beam weapons used by some other species, this laser is subject to the effects of diffraction, causing its damage to become less intense with distance while also rendering it invisible. However, precise optics allow the beam to be targeted from an exceptional range, despite the tenuous damage at the edge of the laser's reach."
+	description "The turret itself only contains the optics needed to synthesize, focus, and target the beam, allowing the laser light to be produced elsewhere. Because of this, an Optical Lasing Generator is needed to run this turret, providing enough power to support a single Blue Laser."
```

Comment:
```suggestion
	description "The turret itself only contains the optics needed to synthesize, focus, and target the beam, requiring the laser light to be produced elsewhere. Because of this, an Optical Lasing Generator is needed to run this turret, providing enough power to support a single Blue Laser."
```